addons/account/wizard/account_payment_register_views.xml

```diff
@@ -118,7 +119,7 @@
                             />
                             <field name="payment_date"/>
                             <field name="communication"
-                                   invisible="not can_edit_wizard or (can_group_payments and not group_payment)"/>
+                                   invisible="not show_amount_div or (can_group_payments and not group_payment)"/>
```

Comment:
?_? so the computed field show_amount_div isn't meant to hide the `<div name='amount_div'> `??  :upside_down_face:

Reply:
You're right twice :D It wasn't supposed to be used here.
But it was supposed to hide the amounts, but after discussions with PO's, we changed the spec.
So the field is in fact useless (so I removed it)